flyteadmin/pkg/repositories/transformers/execution.go

```diff
@@ -128,7 +128,7 @@ func CreateExecutionModel(input CreateExecutionModelInput) (*models.Execution, e
 			Name:    input.WorkflowExecutionID.Name,
 		},
 		Spec:                  spec,
-		Phase:                 input.Phase.String(),
+		Phase:                 closure.Phase.String(),
```

Comment:
should we get rid of Phase in the Input altogether now that we always read it from the closure?

Reply:
Looks like we need phase in execution model to do the filter db query?

Reply:
oh yes we should still set the Phase in the model, I think we just don't need to have it as a field in `CreateExecutionModelInput` if we read it from the closure now

Reply:
Sounds good